test/functional/sessions.test.js

```diff
@@ -66,7 +66,12 @@ describe('Sessions', function () {
     });
 
     describe('withSession', {
-      metadata: { requires: { mongodb: '>=3.6.0' } },
+      metadata: {
+        requires: {
+          mongodb: '>=3.6.0',
+          serverless: 'forbid' // UnhandledPromiseRejectionWarning: MongoServerError: CMD_NOT_ALLOWED: dropAllUsersFromDatabase
```

Comment:
could you explain what this comment is referencing?

Reply:
The command `dropAllUsers`used by that test isn't allowed on serverless, so the test needs to be skipped. Do you think the comment needs to be more descriptive? E.g. "Skipped due to error on serverless: ..."

Reply:
is it the test.setup that does that? `dropAllUsersFromDatabase` doesn't seem essential to sessions (though I may be missing something here), in which case, instead of skipping these tests, it might be better to just refactor the setup to make it compatible with serverless? if that sounds accurate but isn't a quick fix, I think just logging a follow jira ticket to re-enable them later would be fine

Reply:
Took a second look at this, the test actually passes and doesn't need to be skipped.